app/models/manageiq/providers/ansible_tower/inventory_collection_default/automation_manager.rb

```diff
@@ -1,3 +0,0 @@
-class ManageIQ::Providers::AnsibleTower::InventoryCollectionDefault::AutomationManager < ManagerRefresh::InventoryCollectionDefault
```

Comment:
don't we need this one? Or where is the shared module included? Not sure if we need a shared module in this case, since it's ansible specific?